test/commons/color/get-background-color.js

```diff
@@ -259,6 +259,19 @@ describe('color.getBackgroundColor', function () {
 		assert.closeTo(actual.alpha, expected.alpha, 0.1);
 	});
 
+	it('should allow a scroll option', function() {
+		fixture.innerHTML = '<div id="parent" style="height: 40px; width: 30px; ' +
+			'background-color: white; position: relative; z-index: 5">' +
+			'<div id="target" style="position: relative; top: 1px; height: 20px; ' +
+			'width: 25px; z-index: 25;">' + '</div>';
+		var targetEl = fixture.querySelector('#target');
+		var bgNodes = [];
+		var useScroll = false;
+		window.scroll(0, 0);
 
+		commons.color.getBackgroundColor(targetEl, bgNodes, useScroll);
 
+		assert.equal(window.pageYOffset, 0);
+		assert.equal(window.pageXOffset, 0);
+	});
```

Comment:
@ckundo you should probably add a test that tests to ensure that no scrolling occurs when the option is enabled - WDYT?